test/e2e/README.md

```diff
@@ -192,6 +192,39 @@ You can load the new image into the kind cluster using the command below:
 kind load docker-image projects.registry.vmware.com/antrea/antrea-ubuntu:latest --name <kind_cluster_name>
 ```
 
+## Running the e2e tests on existing K8s cluster
+
+If you already have a K8s cluster on which antrea has been pushed, the following steps should be done for running the e2e tests.
+
+First, you should create the `config` file under the default ssh directory and add the ssh information of each node in the cluster to it. You can add the ssh information like the example below:
+
+```bash
+Host Control-Plane-Node
+    HostName <Control-Plane-IP>
+    Port 22
+    user ubuntu
+    IdentityFile /home/ubuntu/.ssh/id_rsa
+Host Worker-Node
+    HostName <Worker-Node-IP>
+    Port 22
+    user ubuntu
+    IdentityFile /home/ubuntu/.ssh/id_rsa
+```
+
+Make sure `Host` of each node must be the same as the corresponding node name. The `Port` is the default port of ssh service. As for the `IdentityFile`, add the location of the private key (`id_rsa` in example) to it.
+
+Beside the configuration of the ssh connection, make sure you have added the public key to `authorized_keys` of each node and set `PubkeyAuthentication` and `RSAAuthentication` of ssh service to `yes`.
+
+Second, when running the e2e tests, the configuration information of cluster will be used, so make sure you have copied the `config` file of the cluster to your localhost under the `/home/<user>/.kube` directory.
```

Comment:
there is aslo option for kubeconfig `--remote.kubeconfig`, you can check the source code here https://github.com/antrea-io/antrea/blob/main/test/e2e/providers/remote.go#L28-L32